airbyte-integrations/bases/base-normalization/entrypoint.sh

```diff
@@ -93,12 +137,16 @@ function main() {
 
   case "$CMD" in
   run)
+    openssh
     configuredbt
     # Run dbt to compile and execute the generated normalization models
     dbt run --profiles-dir "${PROJECT_DIR}" --project-dir "${PROJECT_DIR}"
+    closessh
     ;;
   configure-dbt)
+    openssh
```

Comment:
TODO: doublecheck if this is actually needed here for configuredbt... might not be